### DISCUSSION TITLE
Slowdown on DS games

### DISCUSSION THREAD

**Discussion by istupido** [EmulatorJS/EmulatorJS#825] [General]
So when i play ds games using emulatorjs, I get a slowdown. I think this is because I'm trying to play with my potato Chromebook, so it can't handle it. However, the games can still get 30 fps. Can I get a 30fps framerate without slowing down the game?

**Comment by ethanaobrien** [comment]
You cannot, the frame rate is also tied to the emulation speed

**Reply by noelapplegate** [reply]
Are you using the EmulatorJS Demo? If so, this is because the demo is only to prove the code works, and cannot run the Just-in-Time from a web browser.

**Comment by istupido** [comment]
Is there any other solution to my problem other than buying a better computer

**Reply by ethanaobrien** [reply]
There is not

**Comment by istupido** [comment]
ok thx for the fast reply!

**Comment by istupido** [comment]
what about frame skipping?

**Reply by ethanaobrien** [reply]
Won't likely make a difference, the desmume core supports this option though

**Comment by istupido** [comment]
after doing a few optimizations, i was able to fix my issue.

1. I changed the core to desmume cuz it runs better for me
2. I set frame skipping to 3
3. I set No. of cores to 4

Now I get fps in the 50-60 range in Mario kart ds 

Thanks for the help :)

**Comment by hallohehehehee** [comment]
Games usually get slower on Chromebook when its on battery saving mode
